plinth/utils.py

```diff
@@ -52,6 +52,11 @@ def non_admin_view(func):
     return func
 
 
-def is_user_admin(user):
+def is_user_admin(request, cached=False):
     """Return whether user is an administrator."""
-    return user.groups.filter(name='admin').exists()
+    if cached and request.session['cache_user_is_admin'] is not None:
```

Comment:
`def is_user_admin(request, cached=False):
    """Return whether user is an administrator."""
    if not request.user.is_authenticated():
        return request.user.is_authenticated()

    if 'cache_user_is_admin' in request.session and cached:
        return request.session['cache_user_is_admin']

    user_is_admin = request.user.groups.filter(name='admin').exists()
    request.session['cache_user_is_admin'] = user_is_admin
    return user_is_admin`
Made these changes. I want to test it thoroughly this time though. Can you please tell me where the logging information is stored. I can't seem to find it anywhere, and my terminal never shows anything.